BeOpen-project/BeOpen-Dashboard-Translations
# Update Portuguese Translations to Correct Syntax

### Description

This pull request updates the Portuguese translations in the `pt.json` file to ensure correct syntax and proper terminology. The previous translations contained several inaccuracies and inconsistencies, which have been addressed in this update.

### Changes

- Corrected translations for various labels and messages in the `pt.json` file.
- Ensured consistency and accuracy in the terminology used.

### How to Test

1. Pull the branch locally.
2. Replace the existing `pt.json` file with the updated one.
3. Verify that the translations are accurate and consistent throughout the application.

### Checklist

- [x] Updated translations have been reviewed and confirmed for accuracy.
- [x] Changes have been tested locally to ensure they display correctly.

### Related Issues

N/A

### Screenshots

N/A

### Additional Notes

Please review the updated translations and provide any feedback or further corrections if necessary. Thank you!
